docs/tasks/configure-pod-container/security-context.md

```diff
@@ -23,6 +23,10 @@ a Pod or Container. Security context settings include:
 
 * [Linux Capabilities](https://linux-audit.com/linux-capabilities-hardening-linux-binaries-by-removing-setuid/): Give a process some privileges, but not all the privileges of the root user.
 
+* [AppArmor](https://en.wikipedia.org/wiki/AppArmor): Use program profiles to restrict the capabilities of individual programs.
```

Comment:
nit: maybe link to the kubernetes AppArmor docs instead? https://kubernetes.io/docs/tutorials/clusters/apparmor/